web/Migrations/20220825144120_AddHasGuarantorAndChangeEETInTenants.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class AddHasGuarantorAndChangeEETInTenants : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "NotInEET",
+                table: "Tenant",
+                newName: "InEET");
+
```

Comment:
I think technically to preserve the 'trueness' of our data, we'll want to negate any existing data in the column by adding a line to execute some SQL, like:
```
Sql("UPDATE Tenant SET NotInEET = 1- NotInEET;");
```
@oskarwilliams378 does this look right to you?

Reply:
And we'd need to do the same again in the `Down` method below

Reply:
I'd possibly not do it this way (mathematically), but do it with
```UPDATE Tenant SET InEET = NOT InEET```
I'd also do it with the new column names after we update the column name